src/components/manager/runs/RunLastContent.js

```diff
@@ -112,16 +129,15 @@ const processIngestions = (data, currentTables) => {
                 ? filePaths.map(fileNameFromPath)
                 : [fileNameFromPath(filePaths)];
 
-            const dateStr = getDateFromEndTime(run.details.run_log.end_time);
-            const date = Date.parse(dateStr);
+            const date = Date.parse(run.details.run_log.end_time);
 
-            const currentIngestion = { date: dateStr, dataType, tableId, fileNames };
+            const currentIngestion = { date: date, dataType, tableId, fileNames };
```

Comment:
this can just be `{ date, dataType, ...}` to match the other short forms here
other than that, looks good to me

Reply:
Sure, Thanks!